app/concepts/user/create.rb

```diff
@@ -1,12 +1,11 @@
 class User::Create < Trailblazer::Operation
   include Model
-  model User
-
+  model User, :create
+  
   def process(params)
-    @model = User.new
-    @model.fullname = "Guest"
-    @model.save
-    return @model
+    user = @model = User.new()
```

Comment:
Now that you're using `model User, :create`, this is done for you. `user` is available as `model`.